<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changelogDialog</class>
 <widget class="QDialog" name="changelogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change-Log</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>252</green>
           <blue>250</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>252</green>
           <blue>250</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>134</red>
           <green>129</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="headerLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Log-Entry : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPlainTextEdit" name="logPlainTextEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="previewGroupBox">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>252</green>
           <blue>250</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>130</red>
           <green>125</green>
           <blue>116</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>252</green>
           <blue>250</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>130</red>
           <green>125</green>
           <blue>116</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>130</red>
           <green>125</green>
           <blue>116</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>130</red>
           <green>125</green>
           <blue>116</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="previewPlainTextEdit">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton">
       <property name="text">
        <string>discard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>submitButton</tabstop>
  <tabstop>discardButton</tabstop>
  <tabstop>previewPlainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>submitButton</sender>
   <signal>clicked()</signal>
   <receiver>changelogDialog</receiver>
   <slot>slot_submitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>discardButton</sender>
   <signal>clicked()</signal>
   <receiver>changelogDialog</receiver>
   <slot>slot_discardButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logPlainTextEdit</sender>
   <signal>textChanged()</signal>
   <receiver>changelogDialog</receiver>
   <slot>slot_contentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_contentChanged()</slot>
  <slot>slot_submitButtonClicked()</slot>
  <slot>slot_discardButtonClicked()</slot>
 </slots>
</ui>
